README.md

```diff
@@ -243,26 +243,36 @@ class IterableWeakMap {
     set.delete(ref);
   }
 
-  constructor(iterable) {
-    for (const [key, value] of iterable) {
-      this.set(key, value);
+  constructor(iterable = null) {
+    if (iterable !== null) {
+      for (const { 0: key, 1: value } of iterable) {
+        this.set(key, value);
+      }
     }
   }
```

Comment:
Using object destructuring approximates the [`AddEntriesFromIterable`](https://tc39.es/ecma262/#sec-add-entries-from-iterable) abstract operation more closely.